docs/content/2.features/5.seo.md

```diff
@@ -0,0 +1,57 @@
+# SEO 
+
+## General info 
+
+SEO is an abbreviate for "Search Engine Optimization".
+Exactpro Docs Toolkit has functionality of adding description and keywords for Google search console.
+
+## Configuration
+
+Add `seo` section to `app.config.ts` file of your docs website, using following template: 
+
+```ts
+export default defineAppConfig({
+  exactproDocs: {
+    seo: {
+      description:
+        '1-2 sentences to briefly describe content of your website',
+      keywords: ['keyword1', 'keyword2', 'keyword3'],
+      robots: [{ UserAgent: '*' }, { Allow: '/' }],
+      sitemap: {
+        baseUrl: 'base URL of your website'
+      },
+      verificationMetaTags: [
+        // <meta name="google-search-console-token" />
+        {
+          name: 'google-site-verification',
+          content: 'google-search-console-token'
+        }
+      ]
+    }
+  }
+})
+```
+::notice{note}
+
+You can use `app.config.ts` of our documentation as another example of SEO configuration!
+
+::
+
+## Fields description
+
+| Option | Type     | Description                                                                                                                                  |
+| --- |----------|----------------------------------------------------------------------------------------------------------------------------------------------| 
+| `description` | `string` | Provide 1-2 sentences to briefly describe content of your website. They will be displayed under the link to your website in search results.  |
+| `keywords` | `array`  | Here you should pass all keywords that describes your website as an array, they will be used to characterise your website for search engine. |
+| `robots` | `array`  | Configuration for `robots.txt`.                                      |
+| `sitemap: { baseUrl: }` | `string` | Base URL for sitemap. All links in sitemap will be relative to this URL.                                           |
+| `verificationMetaTags: { name: }` | `string` | Here you should specify name of meta tag, that required for verification of your resource in some search engine.                             |
+| `verificationMetaTags: { content: }` | `string` | Specify content of meta tag as search resource required.                                                                                     |
+
+::notice{note}
+
+See [the page](https://support.google.com/webmasters/answer/9008080?hl=en#meta_tag_verification) about verification meta tags configuration for Google search engine
+
+::
```

Comment:
It is better to place this link right inside table.

Reply:
Moved there